src/lib/components/profile-picture.svelte

```diff
@@ -42,6 +42,23 @@
       destroyInstance();
     }
   }
+  
+  function handleOutsideClick(event: MouseEvent) {
+    if (root && popover) {
+      const target = event.target as Node;
+      if (!root.contains(target) && !popover.contains(target)) {
+        displayPopover = false;
+        destroyInstance();
+      }
+    }
+  }
+
+  onMount(() => {
+    window.addEventListener('click', handleOutsideClick);
+    onCleanup(() => {
```

Comment:
I think the Cleanup-Hook doesn't exist.
You can return a function from `onMount`, that is run to clean up, see the [docs](https://svelte.dev/tutorial/onmount).

Reply:
Hi! Thanks for your input.
I've removed the onCleanup hook and added a return statement.
Please review the code!